src/repo/git.go

```diff
@@ -0,0 +1,68 @@
+package repo
+
+import (
+	"fmt"
+	"github.com/go-git/go-git/v5"
+	"net/url"
+	"os"
+	"strings"
+)
+
+//Clone repo component
```

Comment:
I suggest to remove this comment. 
the comments have official guidline here: https://tip.golang.org/doc/comment
and currently we have nothing to document.
such comments like "Clone does Clone" doesn't bring anything